public/Aryan Suri/Licenses/licenses.js

```diff
@@ -0,0 +1,255 @@
+					break;
+				case "gnudsl":
+					modalC.setAttribute("style", "background-color: #f44336;");
+					modalC.innerHTML = `<span> The content you just copied is ${cc.title} licensed: You have the freedom to run and remix software without profit. Learn more about this license <a href=${cc.link}>here</a> </span>`;
+					break;
+				case "gnufdl":
+					modalC.setAttribute("style", "background-color: #fff176;");
+					modalC.innerHTML = `<span> The content you just copied is ${cc.title} licensed: You have the freedom to run but not remix any software for profit. Learn more about this license <a href=${cc.link}>here</a> </span>`;
+					break;
+				case "arr":
+					modalC.setAttribute("style", "background-color: #f44336;");
+					modalC.innerHTML = `<span> The license of the content on this page All Rights Reserved and the content is allowed to be used on the LibreTexts platform thanks to the author. Usage off the platform requires explicit permission from the content authors.</span>`;
+
+					break;
+				case "cc-publicdomain":
+					return null;
+					break;
+			}
+
+		} else {
+			$(modalB).show();
+			modalC.innerHTML = `<span> The license of the content on this page is unselected. Please review the Contributors and Attributions section or the content author(s) for clarification of the applicable license(s). </span>`
+
+		}
+	}
+
+
+	function ccPageLabel() {
+		const cc = getCC();
+		if (cc) {
+			const pageLabel = document.createElement("li");
+			pageLabel.classList.add("pageInfo");
+			if (cc.label === "gnu") {
+				pageLabel.innerHTML = `<span> <a style="width: max-content; width: -moz-max-content; overflow: initial; font-size:30px" href="${cc.link}"><img style="height: 30px" src="https://awesomefiles.libretexts.org/Social Share Kit/gpl.png" alt="GNU General Public License"/></a>`;
+			}
+			else if (cc.label === "gnudsl") {
+				pageLabel.innerHTML = `<span> <a style="width: max-content; width: -moz-max-content; overflow: initial; font-size:30px" href="${cc.link}"><p style="color: black; font-size: small">GNU Design Science License</p></a>`;
+			}
+			else if (cc.label === "gnufdl") {
+				pageLabel.innerHTML = `<span> <a style="width: max-content; width: -moz-max-content; overflow: initial; font-size:30px" href="${cc.link}"><img style="height: 30px" src="https://awesomefiles.libretexts.org/Social Share Kit/gfdl.png" alt="GNU Free Documentation License"/></a>`;
+			}
+			else if (cc.label === "arr") {
+				pageLabel.innerHTML = `<span> <a style="width: max-content; width: -moz-max-content; overflow: initial; font-size:30px" href="${cc.link}"><img style="height: 30px" src="https://awesomefiles.libretexts.org/Social Share Kit/arr.png" alt="All Rights Reserved"/></a>`;
+			}
+			else {
+				pageLabel.innerHTML = `<span> <a style="width: max-content; width: -moz-max-content; overflow: initial; font-size:30px" href="${cc.link}"><i class='cc ${cc.label}'></i></a>`;
+			}
```

Comment:
Would you be able to update these image links to point to files.libretexts.org? The file paths may also be changed so watch out for that as well.

Reply:
I can do that as a later update for sure